nixos/lib/testing-python.nix

```diff
@@ -0,0 +1,266 @@
+{ system
```

Comment:
Do we need a separate `testing-python.nix` which is mostly identical to `testing.nix`? Maybe we can have a single `makeTest` function that dispatches based on whether the test has a `testScript` or `pythonTestScript` function.

Reply:
In different cases there would be different outcomes:

- keep two test drivers: reduce this duplication
- keep only python driver: remove this file, make the other work with python.

Community discussion results point towards the latter, so i will remove once it's all ported.